### ISSUE TITLE
Add items data file

### ISSUE THREAD

**Issue by thaddeus** [TarkovTracker/tarkovdata#23]
Add an items data file, create an object in an array with a unique id for each unique item utilized in `quests.json` or `hideout.json` along with a 'locales' map with the 'en' field set to the current proper item name. Adding other locales, item shorthands, and filling out items not used for quests or hideout can be added after the fact

**Comment by thaddeus** [comment]
Working on this using the data @kokarn provided in #30 and via his GraphQL API

**Comment by kokarn** [comment]
So it's more of the names than anything else? Do you have a base example of what you had in mind?

**Comment by thaddeus** [comment]
Yeah, it would just be something like:
```
[
	{
		"id": "572b7fa524597762b747ce82",
		"locale": {
			"en": "Item's Proper Name"
		},
		"localeShort": {
			"en": "ShortName"
		}
	},
	{
		"id": "572b7fa524597762b747ce83",
		"locale": {
			"en": "Another Item Name"
		},
		"localeShort": {
			"en": "AnotherShortName"
		}
	},
]
```
Then all of the other metadata about items I'll link via the ID from your API (which I need to play around and learn how to properly form queries for GraphQL, haven't used it before) Stuff like icon, width, and height will all be useful for things I want to add to the tracker (after I finish open sourcing, finishing accounts, live group updates, and probably maps, oh boy there's a lot)

This is the biggest step towards solving the problem of being able to localize all the data other than figuring out how to piece together the data using grammar other than English (no hecking clue how I'm going to figure that out, but hoping for some bilingual people that can advise 😄 )

**Comment by thaddeus** [comment]
Didn't get automatically closed by #31 - closing this now, thank you @kokarn !
